CONTRIBUTING.md

```diff
@@ -8,77 +8,102 @@ Vous trouverez les instructions en français [ici](#contribuer).
 
 Vous êtes tous invités à contribuer à ce projet pour le maintenir ou l'améliorer.
 Même si vous n'êtes pas un développeur, vous pouvez suremnent donner un coup de
-main à la documentation.
-
-### Préparer votre Pull Request (PR)
-
--   Commencez par faire un `fork` de ce dépot.
--   Clonez votre dépot sur votre machine de developpement.
--   Choisissez et installez votre environement de developpement (environement
-    virtuel, container, etc...).
--   Installez le package python en mode edition avec ses prérequis de test:
-    `pip install -e .[testing]`.
--   Créez une branche pour votre contribution.
--   Testez vos modifications avec la commande `pytest`.
--   Créez et envoyez la PR quand elle est prête.
--   Attendez les commentaires des relecteurs, répondez à leur questions ou demandes
-    de mise au point.
--   Votre contribution est fusionnée dans le projet. Merci.
-
-### Bonnes pratiques
-
-Pour faciliter la maintenance et la relecture il est fortement recommandé
-d'utiliser des outils pour s'assurer de la qualité du code et de son formatage.
-Avec les librairies installées par la comande `pip install -e .[testing]`, vous
-pouvez utiliser les commandes:
-
--   `pytest --cov` pour vérifer que les tests couvrent bien tout le code du
-    package.
--   `flake8` pour vérfier que les bonnes pratiques Python sont respectées.
--   `pydocstyle` pour vérifier que les bonnes pratique des docstrings sont
-    respectées.
-
-Vous pouvez utiliser les lignes de commandes ci-dessus ou paramétrer votre
-environnement de developpement pour utiliser ces outils ou similaires.
+main en remontant les bugs constatés, en partageant vos idées d'amélioration ou
+en participant à la documentation.
+
+## Préparer votre environement de développement
+
+Vous aurez besoin de Python 3.6+ et des outils suivants:
+
+-   [Poetry](https://python-poetry.org)
+-   [Nox](https://nox.thea.codes/en/stable)
+
+Installez le package avec les dépendances de développement:
+
+`$ poetry install`
+
+Vous pouvez maintenant utiliser une session interactive Python:
+
+`$ poetry run python`
+
+## Tester le projet
+
+Dérouler la suite de tests complète:
+
+`$ nox`
+
+Lister toutes les sessions disponibles dans Nox:
+
+`$ nox --list-sessions`
+
+Vous pouvez lancer une session Nox spécifique. Par exemple, lancez la suite de
+tests unitaires avec:
+
+`$ nox --session=tests`
+
+Les tests unitaires sont dans le répertoire `tests` et utilisent le framework pytest.
+
+### Soumettre votre Pull Request (PR)
+
+Oouvrez une [pull requet](https://github.com/hacf-fr/meteofrance-api/pulls) pour proposer des changements à ce projet.
```

Comment:
Ouvrez

Reply:
Thanks will be take into account in next PR